angular/core/components/sidebar/sidebar.scss

```diff
@@ -17,42 +28,58 @@
   padding: 0 15px;
   display: block;
   text-align: left;
-  border-radius: 0px;
+  border-radius: 4px;
   text-transform: none;
-  span {
-    font-weight: 600;
-  }
+  font-size: 13px;
+  @include truncateText;
 }
 
 .sidebar__list-item-button--subgroup {
   width: 100%;
   margin: 0;
   text-transform: none;
   display: block;
-  border-radius: 0px;
+  border-radius: 4px;
   text-align: left;
   padding-left: 53px;
+  font-size: 13px;
 }
 
 .sidebar__list-item-icon {
   width: 20px;
   margin-right: 16px;
+  font-size: 22px;
+  color: #666;
+}
+
+.fa-envelope-o {
+  font-size: 20px;
+}
+
+.fa-inbox {
+  color: #5cb85c;
```

Comment:
If you're up for it, I'd suggest defining these as color variables in `0_mixins.scss` (I'm going to merge anyway tho :P)

Also, whenever I see a new 'grey' color, I wonder if we really need it, since we already have ~10 flavours of grey defined in that file.